_rules/audio-or-video-avoids-automatically-playing-audio-80f0bf.md

```diff
@@ -51,20 +53,23 @@ This rule applies to any `audio` or `video` element for which all the following
 - **autoplay**: the element has an `autoplay` [attribute value][] of `true`; and
 - **not muted**: the element has a `muted` [attribute value][] of `false`; and
 - **not paused**: the element has a `paused` [attribute value][] of `false`; and
-- **duration**: the element has a [media resource][] lasting more than 3 seconds and that contains audio.
+- **audio duration**: the element has a [media resource][] for which the audio output lasts more than 3 seconds.
 
-## Expectation
+## Expectation 1
 
-For each test target, the [outcome](#outcome) of at least one of the following rules is passed:
+For each test target, there is at least one [instrument][] in the same [web page][] to pause or stop the audio, or turn the audio volume off independently from the overall system volume control.
 
-- [Audio Or Video That Plays Automatically Has A Control Mechanism](https://act-rules.github.io/rules/4c31df)
-- [Audio Or Video That Plays Automatically Has No Audio That Lasts More Than 3 Seconds](https://act-rules.github.io/rules/aaa1bf)
+## Expectation 2
+
+The [instrument][] to pause or stop or turn the audio volume off is [visible](#visible), has an [accessible name](#accessible-name) that is not only [whitespace](#whitespace), and is [included in the accessibility tree](#included-in-the-accessibility-tree).
```

Comment:
```suggestion
The [instrument][] to pause or stop or turn the audio volume off is [visible](#visible), [included in the accessibility tree](#included-in-the-accessibility-tree), and it has an [accessible name](#accessible-name) that is not only [whitespace](#whitespace).
```
My preference, feel free to ignore.

Reply:
I actually think this expectation should go away in favour of #1632 (where I somehow seem to have forgotten to remove it 😖 )